### PR TITLE
New package: Crunchyroll.MasterofGarden version 1.0.0

### PR DESCRIPTION
**Pull request by solomoncyj**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138731)

### CODE REVIEW

File: manifests/c/Crunchyroll/MasterofGarden/1.0.0/Crunchyroll.MasterofGarden.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Crunchyroll.MasterofGarden
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: Crunchyroll Games
```
Comment (Trenly) [inline manifests/c/Crunchyroll/MasterofGarden/1.0.0/Crunchyroll.MasterofGarden.locale.en-US.yaml:7]:
```suggestion
Publisher: Crunchyroll Games, LLC
```
This should match control panel exactly

File: manifests/c/Crunchyroll/MasterofGarden/1.0.0/Crunchyroll.MasterofGarden.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Crunchyroll.MasterofGarden
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: Crunchyroll Games
+PackageName: 'The Eminence in Shadow: Master of Garden'
```
Comment (Trenly) [inline manifests/c/Crunchyroll/MasterofGarden/1.0.0/Crunchyroll.MasterofGarden.locale.en-US.yaml:8]:
```suggestion
PackageName: MasterofGarden
```
This should match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/c/Crunchyroll/MasterofGarden/1.0.0/Crunchyroll.MasterofGarden.locale.en-US.yaml (hunk 2 above)
Comment (solomoncyj) [reply manifests/c/Crunchyroll/MasterofGarden/1.0.0/Crunchyroll.MasterofGarden.locale.en-US.yaml:8]: Isn’t it supposed to be the full name of the package

File: manifests/c/Crunchyroll/MasterofGarden/1.0.0/Crunchyroll.MasterofGarden.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Crunchyroll.MasterofGarden
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: Crunchyroll Games, LLC
+PackageName: MasterofGarden
```
Comment (stephengillie) [inline manifests/c/Crunchyroll/MasterofGarden/1.0.0/Crunchyroll.MasterofGarden.locale.en-US.yaml:8]:
```suggestion
PackageName: MasterofGarden
PackageUrl: https://www.crunchyroll.com/games/eminenceinshadow/
```

**Comment by solomoncyj** [comment]
Why didn't this pop up on the first run?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**